packages/pwa/app/pages/product-list/index.jsx

```diff
@@ -390,6 +396,14 @@ const ProductList = (props) => {
                                                           : removeItemFromWishlist
                                                       return action(productSearchItem)
                                                   }}
+                                                  imageProps={{
+                                                      sizes: {
+                                                          base: 'calc(100vw / 2)',
+                                                          md: `calc((100vw - ${filtersContainerWidth}) / 3)`,
+                                                          '2xl': '387px'
+                                                      },
+                                                      widths: [400, 800, 1200]
```

Comment:
Is this a true statement base on the configuration that we have above? 

If the screen is determined to be "2xl" based on Chakra responsive layout, we want to use a image with a width of 384px, that will result in the image set image of w=400 based on the browsers internal matching algorithm. If the largest screen size is using the smallest image in the set, why do we have widths of 800 and 1200. Is it because of the pixel density that those smaller screen sizes might have?

Reply:
@bendvc yes, that's right. The 800 and 1200 are for 2x and 3x pixel density screens.

Heads up: I've just updated the widths values. I'll explain how I got to those values in another comment.